### PR TITLE
fix(CommitizenProvider): raise if version is None

### PR DESCRIPTION
**Pull request by bearomorphism**
## Checklist

- [ ] I have read the [contributing guidelines](https://commitizen-tools.github.io/commitizen/contributing/)

### Code Changes

- [ ] Add test cases to all the changes you introduce
- [ ] Run `poetry all` locally to ensure this change passes linter check and tests
- [ ] Manually test the changes:
  - [ ] Verify the feature/bug fix works as expected in real-world scenarios
  - [ ] Test edge cases and error conditions
  - [ ] Ensure backward compatibility is maintained
  - [ ] Document any manual testing steps performed
- [ ] Update the documentation for the changes

### CODE REVIEW

File: commitizen/commands/version.py [hunk 1]
```diff
@@ -28,19 +29,20 @@ def __call__(self) -> None:
...
+            try:
+                version = get_provider(self.config).get_version()
+            except NoVersionSpecifiedError:
                 out.error("No project information in this project.")
-        else:
-            # if no argument is given, show installed commitizen version
-            out.write(f"{__version__}")
+                return
+
+            out.write(("Project Version: " if verbose else "") + version)
```
Comment (Lee-W) [inline commitizen/commands/version.py]:
```suggestion
            out.write(f"Project Version: {version}" if verbose else version)
```

Comment (bearomorphism) [reply commitizen/commands/version.py]: done
